test/integration/integration_test.go

```diff
@@ -118,10 +120,11 @@ func TestIntegration(t *testing.T) {
 }
 
 func TestZoneIntegration(t *testing.T) {
+	readConfigFile(zoneConfigFile)
 	var stop func()
 	ctx := context.Background()
 	fmt.Printf("calling startServer")
-	grpcClient, stop = startServer(ctx, "features/array-config/multi-az")
+	grpcClient, stop = startServer(ctx, zoneConfigFile)
```

Comment:
And here you used zoneConfigFile instead of zoneConfigurationFile which is now inconsistent with your other change.